compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -160,6 +160,56 @@ luci::CircleConst *create_NHWC_from_NCHW(luci::CircleConst *constant)
   return nhwc_const;
 }
 
+// We assume MUL is NCHW if,
+// Input shape: (N, C, H, W)
+// Output shape: (N, C, H, W)
+// 1. Const shape is (1, C, 1, 1)
+// 2. Input, Output, Const have the same C.
+bool is_NCHW_with_const(const luci::CircleMul *node, luci::CircleNode *&pred_node,
+                        luci::CircleConst *&gamma)
```

Comment:
As this is reference to a pointer, how about `luci::CircleConst &*` for `pred_node and `gamma` ?

Reply:
Pointer to reference is not allowed in C/C++.

https://stackoverflow.com/questions/4632762/why-are-pointers-to-a-reference-illegal-in-c

Reply:
> Pointer to reference is not allowed in C/C++.

I know this and wrote `reference to a pointer`. But is `&*` a pointer to reference?

Reply:
I've learned it too. Thanks :smile: